tests/assign/toverload_asgn2.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   output: '''i value 88
 2aa'''
-    knownIssue: "possible issue where result is not treated as var Concrete in passAround"
+  knownIssue.c js vm: "possible issue where result is not treated as var Concrete in passAround"
```

Comment:
```suggestion
  knownIssue: "possible issue where result is not treated as var Concrete in passAround"
```
Since the bug is unrelated to the selected backend (it's a bug in semantic analysis), keeping the `knownIssue` unspecific is a bit clearer.